zipline/assets/assets.py

```diff
@@ -193,7 +197,7 @@ class AssetFinder(object):
     PERSISTENT_TOKEN = "<AssetFinder>"
 
     @preprocess(engine=coerce_string_to_eng)
-    def __init__(self, engine):
+    def __init__(self, engine, future_chain_predicates=CHAIN_PREDICATES):
```

Comment:
Thinking more, this feels like it doesn't quite belong as a parameter to `AssetFinder`, but I'm not sure how else to refactor it. Maybe make this an optional parameter to `get_ordered_contracts`?